app/src/main/java/com/poupa/vinylmusicplayer/helper/menu/MenuHelper.java

```diff
@@ -0,0 +1,28 @@
+package com.poupa.vinylmusicplayer.helper.menu;
+
+
+import android.content.Context;
+import android.text.SpannableString;
+import android.text.style.ForegroundColorSpan;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.poupa.vinylmusicplayer.R;
+
+
+public class MenuHelper {
+
+    public static void setDeleteMenuItemRed(Menu menu, Context context) {
+        // All delete element inside of menu should have red text to better differentiate them
+        MenuItem liveItem = menu.findItem(R.id.action_delete_playlist);
+        if (liveItem == null)
+            liveItem = menu.findItem(R.id.action_delete_from_device);
+
+        if (liveItem != null) {
```

Comment:
At this point, liveItem is not null anymore due to the previous line, right? or could findItem return null?

Reply:
depending on the menu, the delete action as different naming, thus previous line can return nothing